src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/decode/ValueDecoder.scala

```diff
@@ -50,7 +51,7 @@ private[decode] object ValueDecoder {
   implicit final val stringColumnDecoder: ValueDecoder[String] =
     fromFunc[String] {
       case (key, value) =>
-        if (value.isEmpty) (key, s"Field $key cannot be empty").asLeft else value.asRight
+        if (value.isEmpty) InvalidValue(key.toString, s"Field $key cannot be empty").asLeft else value.asRight
```

Comment:
Second member of `InvalidValue` is `value`, but we add a human-readable message here.

Reply:
Yeah I think I missed it, will correct it.